server/queue/queue-pulsar/src/test/java/org/apache/james/queue/pulsar/PulsarMailQueueTest.java

```diff
@@ -316,4 +324,58 @@ public void browseShouldReturnEmptyWhenSingleDequeueMessageEvenWhenStoreIsGuaran
                 .isEmpty();
     }
 
+    @Test
+    void dequeueShouldAllowToNackAndRetrieveSeveralMails() throws Exception {
```

Comment:
ignore this test it was used to investigate the issue and should have been removed